array-methods.js

```diff
@@ -7,6 +9,14 @@ var dataset = require('./dataset.json');
 */
 var hundredThousandairs = null;
 
+function ovaHunned(element) {
+  if (element.amount > 100000) {
+    return element;
+  }
+}
+
+hundredThousandairs = dataset.bankBalances.filter(ovaHunned)
```

Comment:
Although this is completely fine, I would recommend trying to do this with an inline function since that's what we'll be seeing a lot of in the future.